lms/djangoapps/onboarding/models.py

```diff
@@ -253,6 +253,14 @@ def admin_info(self):
         """
         return "%s" % self.admin.email if self.admin else "Administrator not assigned yet."
 
+    @property
+    def first_learner(self):
+        """
+        :return: organization's first learner
+        """
+        org_first_learner = UserExtendedProfile.objects.filter(organization=self, is_first_learner=True).first()
+        return org_first_learner.user if org_first_learner else None
+
     def get_active_partners(self):
         """ Return list of active organization partners"""
         return self.organization_partners.filter(end_date__gt=datetime.utcnow()).values_list('partner', flat=True)
```

Comment:
```
org_first_learner = self.extended_profile.filter(is_first_learner=True).first()
if org_first_learner:
    return org_first_learner.user
```

Please see the same kind of thing that I suggested earlier and pasted above is used in `get_active_partners` method. **Organization** model doesn't have `organization_partners` as well. But its reverse relation. 
You can see the **OrganizationPartner** model, It has following relative field with **Organization**
`organization = models.ForeignKey(Organization, related_name='organization_partners')`

The point needs to notice is **related_name** that is **organization_partners**. Which is used in Organization Model to get partners and filter accordingly. 

Same is the case with **UserExtendedProfile**. You can see the following field in **UserExtendedProfile** model
```
organization = models.ForeignKey(Organization, related_name='extended_profile', blank=True, null=True,
                                     on_delete=models.SET_NULL)
```
It has related_name **extended_profile** which we can use in this method. 
Please share your thoughts on it?

Reply:
Yes @zeshanarifios  bhai this worked (Y) i have changed the code and tested it this working fine.

Reply:
@zahid-arbisoft 👆